lib/plugins/aws/package/compile/events/apiGateway/lib/authorizers.js

```diff
@@ -37,7 +37,7 @@ module.exports = {
               '/invocations',
             ],
           ] };
-          authorizerProperties.Type = 'TOKEN';
+          authorizerProperties.Type = authorizerProperties.Type ? authorizerProperties.Type.toUpperCase() : 'TOKEN';
```

Comment:
- Linting error occurs with the maximum line length of 100, so you might want to add the line break to the right position.

- Could you change authorizerProperties.Type --> authorizerProperties.type since we follow camelCase as the coding rule.

Reply:
Thanks for commenting @horike37 👍

> Could you change authorizerProperties.Type --> authorizerProperties.type since we follow camelCase as the coding rule.

Not sure if this is necessary in that case since it was uppercase before the change as well. We definitely need to check that the configuration still works if we do that change.